script.service.ltvpl/resources/lib/Utilities/VirtualEvents.py

```diff
@@ -0,0 +1,47 @@
+#VirtualEvents.py
+
+try:
+    from Tkinter import *
```

Comment:
I hope this is only for compatibility for environments outside of kodi

Reply:
You're correct with your assumption. This back end code also runs under Windows.